src/api/geocoder.ts

```diff
@@ -46,7 +46,7 @@ export async function getFeatureFromVenue(
       lon: venue.coordinates.longitude,
       limit: 1,
       layers: ['venue'],
-      tariff_zone_authorities: 'ATB',
```

Comment:
To be more precise, "AtB" is not a tariff zone, but the authority (organization) for the tariff zone collection. Maybe use `TARIFF_ZONE_AUTHORITY` for the env variable as well.

Reply:
Ah, yes, will rename to `TARIFF_ZONE_AUTHORITY` 👍